### DISCUSSION TITLE
无法安装v4.6.0版本怎么回事？

### DISCUSSION THREAD

**Discussion by oomochi** [ywmoyue/biliuwp-lite#93] [Q&A] [answered]
安装和更新都显示“应用安装程序无法安装应用包相关项。请向开发人员索要 应用包“，之前那两个测试版4.6.0.1和4.6.0.2也是这样

**Comment by ywmoyue** [comment, marked as answer]
1. 检查应用商店的下载列表中是否有相关依赖项未成功安装，如果有，则可能是网络问题需要您自行解决
2. 尝试该方法https://github.com/ywmoyue/biliuwp-lite/discussions/60#discussioncomment-5759801

**Reply by oomochi** [reply]
感谢，已正常安装
